### DISCUSSION TITLE
I would request code owners

### DISCUSSION THREAD

**Discussion by lnovizio** [community/community#23612] [New to GitHub]
<p>can you tell me where I can find the code owners of my repository?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>check this out,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/f/4f94ce56c2b644c230e145c24e4f04d19fe73c6e.png" data-download-href="/uploads/short-url/bm0xcKo7AJtEOrAgspsZm8pymeO.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078226-a4d31678-4941-4c7f-a106-f627771e1f28.png" alt="image" data-base62-sha1="bm0xcKo7AJtEOrAgspsZm8pymeO" width="690" height="309" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/f/4f94ce56c2b644c230e145c24e4f04d19fe73c6e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">773×347 24.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p><a href="https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners">https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners</a></p>

**Reply by jkeehley-boop** [reply]
I'm really dumb at computers. I won 106,000.10 on bonus play. It won't let me link my phone. And I can't find the code  so I can withdraw from it. While in return the game will give me $777.00 in return
